packages/sdk/src/modules/base/index.ts

```diff
@@ -213,30 +231,32 @@ export class Base {
                       unsubscribeNewHeads();
                     }
 
-                    const eventsTriggeredByTx: any = []
```

Comment:
previous event log did not work. Updated to fix and now resolves to peaqEvents with data